src/views/dialog.rs

```diff
@@ -14,8 +14,13 @@ use view::{AnyView, Selector, View};
 use views::{Button, DummyView, SizedView, TextView};
 
 #[derive(PartialEq)]
-enum Focus {
+/// Identifies currently focused element in [`Dialog`].
+///
+/// [`Dialog`]: struct.Dialog.html
+pub enum DialogFocus {
```

Comment:
Let's add the usual `#[derive(Clone,Copy,Debug,PartialEq,Eq)]`, since it's now a public field.

Reply:
done